Sources/DataConnect.swift

```diff
@@ -97,12 +111,14 @@ public class DataConnect {
     self.connectorConfig = connectorConfig
     self.callerSDKType = callerSDKType
 
+    let auth = Auth.auth(app: app)
+    nonisolated(unsafe) let appCheck = AppCheck.appCheck(app: app)
     grpcClient = GrpcClient(
       app: self.app,
       settings: settings,
       connectorConfig: connectorConfig,
-      auth: Auth.auth(app: app),
-      appCheck: AppCheck.appCheck(app: app),
+      auth: auth,
+      appCheck: appCheck,
```

Comment:
Since this class doesn't appear to retain auth or app check instance, maybe delay calling `Auth.auth(app: app)` and `AppCheck.appCheck(app: app)` in this scope, instead get them in the `GrpcClient` initializer. Perhaps this will prevent the need for the `nonisolated(unsafe) let appCheck = AppCheck.appCheck(app: app)` since the app check instance isn't being passed out of scope.

Reply:
That change worked and resolved the nonisolated requirement in the outer class. Thanks!